swabbie-web/src/main/kotlin/com/netflix/spinnaker/swabbie/controllers/ResourceController.kt

```diff
@@ -71,13 +70,33 @@ class ResourceController(
   }
 
   @RequestMapping(value = ["/canDelete"], method = [RequestMethod.GET])
-  fun markedResourcesReadyForDeletion(): List<MarkedResource> = resourceTrackingRepository.getMarkedResourcesToDelete()
+  fun markedResourcesReadyForDeletion(): List<MarkedResource> =
+    resourceTrackingRepository.getMarkedResourcesToDelete()
+
+  @RequestMapping(value = ["/states/{status}"], method = [RequestMethod.GET])
+  fun getResourcesByState(
+    @PathVariable status: String
```

Comment:
Did you try making the `status` value the `Action` type? Spring is smart enough to handle enums on its own.

Reply:
Not totally sure what `"FAILED"` is about... seems we're conflating status and actions as a single value?

Reply:
Did not try. 

With FAILED - trying to record when an action has failed on the orca side with good visibility, but there is no opportunity for a message field on the Status object I'm leaning towards adding an optional message on the status object so that it can be in state "FAILED" with a message about what action failed. Any thoughts on how to not make this feel weird?

Reply:
Will address this in a different PR.